### DISCUSSION TITLE
Is it possible to do a single FTP upload with libcurl in a couple of threads?

### DISCUSSION THREAD

**Discussion by ribtoks** [curl/curl#11518] [Q&A] [answered]
e.g. FileZilla is uploading 1 file in 2 threads. Is it possible to do the same in curl? (possibly with a curl handle reuse)

**Comment by bagder** [comment, marked as answer]
No, it's not possible. And I also wonder how on earth FileZilla does it, since an FTP server won't allow that without some magic hackery applied..

**Reply by ribtoks** [reply]
Thank you for your response.
